recipe/meta.yaml

```diff
@@ -16,10 +16,10 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3
```

Comment:
could we add comments on these describing that they're there in order to drop the futures py2-only dependency and not have selectors in the recipe?